_includes/pbc/all/install-features/202407.0/install-the-spryker-core-back-office-feature.md

```diff
@@ -0,0 +1,751 @@
+{% info_block errorBox %}
+
+This feature installation guide expects the basic feature to be in place. This guide adds the following functionalities:
```

Comment:
Should this list contain something related to audit logs?

Reply:
Fixed